bundles/net.tourbook/src/net/tourbook/ui/views/nutrition/DialogSearchProduct.java

```diff
@@ -0,0 +1,1078 @@
+
+   private static final String           ID                                             = "net.tourbook.ui.views.nutrition.DialogSearchProduct"; //$NON-NLS-1$
+
+   private static final IPreferenceStore _prefStore                                     = TourbookPlugin.getPrefStore();
+   private static final IDialogSettings  _state                                         = TourbookPlugin.getState(ID);
+   private static final String           STATE_SEARCHED_QUERIES                         = "searched.queries";                                    //$NON-NLS-1$
+   private static final String           STATE_SEARCH_TYPE                              = "STATE_SEARCH_TYPE";                                   //$NON-NLS-1$
+   private static final String           COLUMN_CODE                                    = "Code";                                                //$NON-NLS-1$
+   private static final String           COLUMN_NAME                                    = "Name";                                                //$NON-NLS-1$
+   private static final String           COLUMN_QUANTITY                                = "Quantity";                                            //$NON-NLS-1$
+
+   private static final String           HTTPS_OPENFOODFACTS_PRODUCTS                   = "https://world.openfoodfacts.org/cgi/product.pl";      //$NON-NLS-1$
+
+   private TableViewer                   _productsViewer;
+   private List<Product>                 _products;
+   private ProductComparator             _productComparator                             = new ProductComparator();
+
+   private long                          _tourId;
+
+   private boolean                       _isInUIInit;
+
+   /*
+    * none UI
+    */
+   private PixelConverter          _pc;
+   private List<String>            _searchHistory                  = new ArrayList<>();
+   private final NutritionQuery    _nutritionQuery                 = new NutritionQuery();
+   private PostSelectionProvider   _postSelectionProvider;
+
+   private IPropertyChangeListener _prefChangeListener;
+   private SelectionListener       _columnSortListener;
+
+   private AutocompleteComboInput  _autocompleteProductSearchHistory;
+
+   private ControlDecoration       _decorator_InvalidBarCode;
+
+   private ComboViewer             _queryViewer;
+
+   private ColumnManager           _columnManager;
+   private MenuManager             _viewerMenuManager;
+   private IContextMenuProvider    _tableViewerContextMenuProvider = new TableContextMenuProvider();
+
+   /*
+    * UI controls
+    */
+   private Button                    _btnAdd;
+   private Button                    _btnSearch;
+
+   private Label                     _lblKeywords;
+   private Label                     _lblSearchType;
+
+   private Combo                     _comboSearchQuery;
+   private Combo                     _comboSearchType;
+
+   private Composite                 _viewerContainer;
```

Comment:
Now the "Alignment" column is:
- not by default visible
- the last column

Label and tooltip updated per your suggestion

Reply:
It would be nice that all the viewers in MT are consistent and have this "Alignment" column.

This is the Marker View as of 24.1 for example:
![image](https://github.com/mytourbook/mytourbook/assets/25235665/962de481-48d2-4df9-88a7-d7f7732e911c)

Reply:
I just discovered that there was already an "Alignment" force column, years ago. 

The first visible column in the time slice viewer is right aligned and used the same trick with a hidden column with a `width = 0`

Reply:
>I just discovered that there was already an "Alignment" force column, years ago.

I don't see where this "old" column from years ago is in MT

Reply:
![image](https://github.com/mytourbook/mytourbook/assets/1283445/5f778d5e-d1af-43a9-937a-efcdcc6300e4)

Reply:
Ok, I didn't see that one but now i see it